.gitignore

```diff
@@ -150,3 +150,6 @@ data/
 plots/
 results/
 result_metrics/
+
+#React Folders/Files to be Ignored
+/node_modules
```

Comment:
Needed to add `/node_modules` to the `.gitignore` file because these are auto generated once a user runs the react application for the first time

Reply:
Thanks for the PR, Adi. This looks a lot more manageable. Could you add as a comment in the git-ignore and in the README that these are autogenerated?